tools/webinstall/install.sh

```diff
@@ -1713,7 +1713,9 @@ main() {
     _main_kraft_ret="$?"
 
     # Install kraftkit completions
-    install_completions "$_main_arch"
+    if [ "$NEED_TTY" != "n" ]; then
```

Comment:
```suggestion
    if [ "$NEED_TTY" = "y" ]; then
```

Let's invert the check to make sure that we respect the default value `y`.

We need to do the check here because there is no default value for the shell type.